engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -121,7 +125,7 @@ public void processRecord(
     final var process =
         processState.getProcessByKey(processInstanceRecord.getProcessDefinitionKey());
 
-    final var validationResult = validateCommand(command, process);
+    final var validationResult = validateCommand(command, process, processInstance);
```

Comment:
🔧 We don't need the `processInstance` here. The only reason we are using it is for getting the process id, which is already available in the `process`.